src/pray.c

```diff
@@ -1554,8 +1554,11 @@ dosacrifice()
             } else { /* super big win */
                 adjalign(10);
                 u.uachieve.ascended = 1;
-                pline(
-               "An invisible choir sings, and you are bathed in radiance...");
+                /* you might be able to see the invisible choir! */
+                pline("%s sings, and you are bathed in radiance...",
+                      Hallucination ? "The fat lady"
```

Comment:
"The fat lady sings" is arguably a form of fat shaming. Whether you personally agree or not, maybe it's not something to subject a player to in their moment of triumph.

Maybe a reference to psychedelic rock instead? Perhaps a low-key Grateful Dead reference?

Reply:
"It's not over till the fat lady sings" is a proverb, and meshes well with the usual message involving singing and it taking place at the end of the game. I don't consider this to be fat shaming or fatphobic because it's not directed at anyone, isn't casting shame on anyone, and isn't normalizing non-fat people.

Reply:
I totally get that it's a proverb and think that you are operating with good intentions here, but I'm not really trying to argue that it is or isn't offensive, specifically. What I am arguing is that there are a lot of people who do think it's offensive, which you can confirm with some web searches, and that that position is reasonable, and that it would be irritating to such a person to subject them to what they consider to be an offensive joke right when they should be celebrating.

And like I said there are other jokes to make. It wouldn't even be the first Grateful Dead reference, see https://github.com/NetHack/NetHack/blob/NetHack-3.7/dat/epitaph.txt#L211.

Reply:
Sorry, no interest in changing it. Getting offended by a reference to a proverbial (not real) fat lady is an unreasonable response and might indicate that the reader needs to grow a thicker skin.

If one applies this logic elsewhere, one finds oneself unable to describe anyone as fat, in any fiction, ever, because someone somewhere might be too sensitive to read something with the barest hint of an implication that "fat" can be used as a distinguishing adjective on a person.

Reply:
I agree with @copperwater . Pretty much anyone could be offended with pretty much anything, even the most innocent things. Despite the world becomes a strange place, where people are forced to accept as norm to say "sorry" for something that never had no intention to insult anyone, we should try to maintain some sanity for sake of mankind and it's future.

I mean, some feminists are offended with the existence of men itself and they find males offending just because they exist at all. You wouldn't commit suicide just because your existence _might_ offend some feminists, would you? Some people think that "whitelist" and "blacklist" and "master-slave" (in technical context) supports white supremacy and slavery, despite (in technical speak) they're never meant to convey anything like that. And instead of solving real problems (slavery is still a thing, but of course, fighting against the words like "blacklist" is much easier than actually freeing enslaved people), childfree people are offended with the existense of children, despite they also were kids some time ago (what a hypocrisy!).

If we continue down the road of banning words just because they might be offending to somebody - we would eventually run out of vocabulary one day.

Reply:
This doesn't need to be a big deal. No one is demanding an apology. This is a new change, not some 30 year old piece of NetHack history. The "joke" itself isn't that funny. Reasonable people can and do disagree on whether "the fat lady sings" is offensive/annoying/whatever. We don't need to try out edgy humor (edgy to other people, not necessarily to you) on people as they're crossing the finish line of a NetHack marathon.

We can come up with something else. I propose something like "You hear Jimi Hendrix playing, and are bathed in psychedelic stage lights...." I'm sure there are many, many other alternatives.

Reply:
@jeremyn whatever you're smoking, I want it so I can forget ever reading this